modules/contact/test/tests/multiple_contact_pairs/three_hexagons_coarse.i

```diff
@@ -52,7 +54,7 @@
     [hex1_pressure]
       boundary = '110'
       function = pressure
-      factor = 140
+      factor = 80
```

Comment:
Why this change?

Reply:
The framework changes in this PR affect the results of this test. This particular change reduces the amount of deformation which speeds up the results. Since the purpose of these tests is to check for the action correctness, I think it's fine.